tests/providers/amazon/aws/transfers/test_s3_to_redshift.py

```diff
@@ -443,8 +492,21 @@ def test_using_redshift_data_api(self, mock_rs, mock_run, mock_session, mock_con
             StatementName=statement_name,
             WithEvent=False,
         )
+
+        expected_copy_query = """
+                        COPY schema.table
+                        FROM 's3://bucket/key'
+                        credentials
+                        'aws_access_key_id=aws_access_key_id;aws_secret_access_key=aws_secret_access_key'
+                        ;
+                     """
+        actual_copy_query = mock_rs.execute_statement.call_args.kwargs["Sql"]
+
         mock_rs.describe_statement.assert_called_once_with(
             Id="STATEMENT_ID",
         )
+
+        assert access_key in actual_copy_query
+        assert secret_key in actual_copy_query
         # test sql arg
```

Comment:
this comment seems to be redundant since the next statement is clear on what it's doing.